cmd/cmd.js

```diff
@@ -180,19 +180,21 @@ class Cmd {
       .action(function(env, options) {
         checkDeps();
         i18n.setOrDetectLocale(options.locale);
-        const nullify = (v) => (!v || typeof v !== 'string') ? null : v;
+        const nullify = (v) => {
+          return (!v || typeof v !== 'string') ? null : v;
+        };
         embark.run({
           env: env || 'development',
           serverPort: nullify(options.port),
           serverHost: nullify(options.host),
           client: options.client || 'geth',
           locale: options.locale,
-          runWebserver: options.noserver == null ? null : !options.noserver,
+          runWebserver: options.noserver === null ? null : !options.noserver,
```

Comment:
The `==` here was on purpose, to test for `undefined` or `null`. When one side of `==` is `null`, the other [can only be](https://dorey.github.io/JavaScript-Equality-Table/) `undefined` or `null` for the expression to be true.

Reply:
Oh, it's not quite clear, because == is not clear. I'll add a && so it makes more sense

Reply:
I meant `||` actually. I panned on using `options.noserver === null || options.noserver === undefined`, but looks like `!options.noserver` works too. But when putting noserver, something crashes (not due to this change). I'm investigating.

Reply:
Actually `options.noserver === null || options.noserver === undefined` is better if `== null` isn't acceptable. We don't want to pass `undefined` to cmd_controller because it's setup to use `null` as a signal value.

`== undefined` and `== null` (or either flipped around) are two okay uses (the only two, really) of `==` because the rule is simple — there is't anything else that can `==` null/undefined but null/undefined.